Cargo.toml

```
@@ -22,8 +22,9 @@ path = "tests/lib.rs"
 test = true
 
 [features]
-default = ["build", "git"]
+default = ["build", "git", "validation"]
 build = ["tonic-build", "protoc-bin-vendored"]
```

```suggestion
build = ["dep:tonic-build", "dep:protoc-bin-vendored"]
```

to keep it symmetrical